README.md

```diff
@@ -88,6 +92,38 @@ $ export updates_repo_url="<updates_repo_url_to_package_updates>"
 $ make rhel_iso
 ```
 
+<a name="releasing-minishift-iso"></a>
+## Releasing Minishift ISO
+
+<a name="create-a-release-notes-pr"></a>
+### Create a Release Notes PR
```

Comment:
Also, imo there is no need for a release note pull request. In fact I am happy to skip the release notes completely at least for now.